src/Nethermind/Nethermind.Consensus/Processing/OneTimeProcessor.cs

```diff
@@ -39,17 +39,7 @@ public Task StopAsync(bool processRemainingBlocks = false)
         {
             lock (_lock)
             {
-                Block result;
-                try
-                {
-                    result = _processor.Process(block, options, tracer);
-                }
-                finally
-                {
-                    _worldState.Reset();
-                }
-
-                return result;
+                return _processor.Process(block, options, tracer);
```

Comment:
hmmm but on version 1.24 we used to call `_readOnlyDbProvider.ClearTempChanges();`
https://github.com/NethermindEth/nethermind/blob/release/1.24.0/src/Nethermind/Nethermind.Consensus/Processing/OneTimeProcessor.cs